docs/docs/en/guide/parameter/context.md

```diff
@@ -76,3 +76,21 @@ The result of Node_mysql is as follows:
 Even though output is assigned a value of 1 in Node_A's script, the log still shows a value of 100. But according to the principle from [parameter priority](priority.md): `Local Parameter > Parameter Context > Global Parameter`, the output value in Node_B is 1. It proves that the output parameter is passed in the workflow with reference to the expected value, and the query operation is completed using this value in Node_mysql.
 
 But the output value 66 only shows in the Node_A, the reason is that the direction of value is selected as IN, and only when the direction is OUT will it be defined as a variable output.
+
+### HTTP
+
+Step 1: Drag an HTTP task, fill IN body for KEY, select OUT for IN/OUT, select VARCHAR for output data type, always select VARCHAR, nothing else.
+
+![httpParam-1](../../../../img/httpParam/httpParam-1.png)
+
+Step 2: After adding an HTTP task type node, accept the parameters passed upstream. This time just add it in the Request Parameters section,
+
+![httpParam-2](../../../../img/httpParam/httpParam-2.png)
+
+It can be anything, it can be body, it can be anything, parameter,value, it must be the key that you pass upstream. In this case, the key that you pass upstream is body, so we use ${body}
+
+The configuration is complete
+![httpParam-3](../../../../img/httpParam/httpParam-3.png)
+
+Step 3: You can write a test interface to test whether our parameters are passed successfully.
+![httpParam-4](../../../../img/httpParam/httpParam-4.png)
```

Comment:
I think we don't need this in doc, WDYT? cc @zhongjiajie 

BTW, if add, it's better to use English log.

Reply:
It has been processed and the code has been committed